qutebrowser/browser/webengine/webview.py

```diff
@@ -144,6 +147,20 @@ def _set_bg_color(self):
             col = self._theme_color
         self.setBackgroundColor(col)
 
+    @pyqtSlot(QUrl, 'QAuthenticator*', 'QString')
+    def _on_proxy_authentication_required(self, url, authenticator,
```

Comment:
Now you don't use `url` anymore, so that should be `_url` (see [pylint](https://travis-ci.org/qutebrowser/qutebrowser/jobs/276727224#L471))